changelog.d/pa-3262.fixed

```diff
@@ -0,0 +1,14 @@
+In expression-based languages, definitions are also expressions.
+
+This change allows dataflow to properly handle definition expressions.
+
+For example, the pattern `0 == 0` will match `x == 0` in
+
+```elixir
```

Comment:
Maybe add this example as a test too?

Reply:
Sure, oh, now I'm reminded, that I should add the test in pro, including the one in this PR too.

Reply:
Actually, the taint maturity tests are still here. I'll add a test here then and move it later.